app/public/css/main.css

```diff
@@ -68,8 +68,11 @@ html, body {
 */
 .slide-up-tab {
   position: absolute;
+  z-index: 100;
+  top: 10%;
+height: 85%;
```

Comment:
Practicality comes before aesthetics! But I'll do it anyway since that's probably the last thing now. Boy, you sure care about indentation, putting it as the first comment and all